controller/src/main/java/com/epam/esm/controller/exception/RestExceptionHandler.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.esm.controller.exception;
+
+import com.epam.esm.model.service.exception.ServiceException;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RestControllerAdvice;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+@RestControllerAdvice
+public class RestExceptionHandler {
+
+    private final AtomicInteger atomicInteger = new AtomicInteger();
+
+    @ExceptionHandler(ServiceException.class)
+    private ResponseEntity<ErrorResponse> handleException(ServiceException exception) {
+        HttpStatus status = HttpStatus.NOT_FOUND;
+        ErrorResponse errorResponse =
```

Comment:
It looks really strange with AtomicInteger and mathematic operations :)

Reply:
it is what it is